ui/src/project.ts

```diff
@@ -198,3 +203,92 @@ export const fetchList = event.create<Kind, Msg>(Kind.FetchList, update);
 
 // Fetch initial list when the store has been subcribed to for the first time.
 projectsStore.start(fetchList);
+
+// NEW PROJECT
+
+export const localState = writable<LocalState | string>("");
```

Comment:
We usually don't export the writables. Rather we attempt to derive a readable and have mutations done through functions/events.

Reply:
We set up the `readable` for remote stores. These stores are more simple and we export simple `writables` throughout the app. Could you explain why we would want to avoid exporting them?

Reply:
Mostly for semantics and trace of write patterns as in who are the writers to this state. We don't need to go all out on this for this at the moment.